shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FaultParameters.java

```diff
@@ -19,11 +19,18 @@
 import java.util.List;
 import java.util.Objects;
 
+import static com.powsybl.shortcircuit.ShortCircuitConstants.InitialNominalVoltageMapType;
+import static com.powsybl.shortcircuit.ShortCircuitConstants.InitialVoltageMapType;
+
 /**
  * @author Thomas Adam <tadam at silicom.fr>
  */
 public class FaultParameters {
 
+    // VERSION = 1.0 withLimitViolations, withVoltageMap, withFeederResult, studyType and minVoltageDropProportionalThreshold
+    // VERSION = 1.1 initialVoltageMapType, initialNominalVoltageMapType
```

Comment:
Can you, in order to ease the understanding of your work, also change in that PR the word `Map` into `Profile`? The parameter `withVoltageMap` will become `withVoltageProfileResult`. Then if we are logical, `initialVoltageMapType` and `initialNominalVoltageMapType` could be `voltageProfileInitializationType` and `nomialVoltageProfileInitializationType`. What do you think?